web/crux/src/app/notification/notification.service.ts

```diff
@@ -32,18 +27,53 @@ export default class NotificationService {
     private httpService: HttpService,
   ) {}
 
-  async createNotification(
-    request: CreateNotificationRequest,
-    identity: Identity,
-  ): Promise<CreateNotificationResponse> {
+  async getNotifications(identity: Identity): Promise<NotificationDto[]> {
+    const notifications = await this.prisma.notification.findMany({
+      where: {
+        team: {
+          users: {
+            some: {
+              userId: identity.id,
+              active: true,
+            },
+          },
+        },
+      },
+      include: {
+        events: true,
+      },
+    })
+
+    const userIds = notifications.map(r => r.createdBy)
+    const identities = await this.kratos.getIdentitiesByIds(new Set(userIds))
```

Comment:
I think there is a getIdentityById()

Reply:
The point of this is to avoid getting the same identity multiple times, so it should be ok like this.

Reply:
my bad